LegalWebsitesTR.js

```diff
@@ -0,0 +1,648 @@
+					item.publisher = productData.brandName;
+				}
+
+				// Get barcode as ISBN if not already found
+				if (productData.barkod && !item.ISBN) {
+					item.ISBN = productData.barkod;
+				}
+			} catch (e) {
+				// Ignore JSON parsing errors
+			}
+		}
+
+		// Get description from meta tag
+		var descMeta = doc.querySelector('meta[name="description"]');
+		if (descMeta) {
+			item.abstractNote = descMeta.getAttribute('content');
+		}
+
+		// Set default language to Turkish
+		item.language = 'tr';
+	}
+	
+	// Get product description for notes field
+	var descriptionElem = doc.querySelector('.product-description');
+	if (descriptionElem) {
+		item.notes.push({note: descriptionElem.textContent});
+	}
+	
+	// Get authors from the author links
+	var authors = doc.querySelectorAll('.hm-product-author-top a');
+	for (var i = 0; i < authors.length; i++) {
+		var creator = cleanCreatorTitles(authors[i].textContent);
+		item.creators.push(ZU.cleanAuthor(creator, "author"));
+	}
+
+	// Look for translators
+	var translatorSection = doc.querySelector('.hm-product-cevirmen-top');
+	if (translatorSection) {
+		var translators = translatorSection.querySelectorAll('a');
+		for (let i = 0; i < translators.length; i++) {
+			let creator = cleanCreatorTitles(translators[i].textContent);
+			item.creators.push(ZU.cleanAuthor(creator, "translator"));
+		}
+	}
+	
+	// Look for editors
+	var editorSection = doc.querySelector('.hm-product-editor-top');
+	if (editorSection) {
+		var editors = editorSection.querySelectorAll('a');
+		for (let i = 0; i < editors.length; i++) {
+			let creator = cleanCreatorTitles(editors[i].textContent);
+			item.creators.push(ZU.cleanAuthor(creator, "editor"));
+		}
+	}
+	
+	// Get edition
+	var editionElem = doc.querySelector('.hm-product-baski-top');
+	if (editionElem) {
+		let editionText = editionElem.textContent;
+		let match = editionText.match(/(\d+)\. Baskı/);
+		if (match && match[1] && match[1] !== "1") {
+			item.edition = match[1];
+		}
+	}
+	
+	// Set language (default to Turkish)
+	item.language = "tr";
+	
+	// Get publication date
+	var dateElem = doc.querySelector('.hm-product-basim-tarihi-top');
+	if (dateElem) {
+		let dateText = dateElem.textContent;
+		let match = dateText.match(/(\d{4})/);
+		if (match && match[1]) {
+			item.date = match[1];
+		}
+	}
+	
+	// Get number of pages
+	var pagesElem = doc.querySelector('.hm-product-sayfa-top');
+	if (pagesElem) {
+		let pagesText = pagesElem.textContent;
+		let match = pagesText.match(/(\d+)\s+sayfa/);
+		if (match && match[1]) {
+			item.numPages = match[1];
+		}
+	}
+	
+	// Handle legal.com.tr specific extraction
+	if (_url.includes('legal.com.tr')) {
+		// Get title from h1.headline
+		var titleElem = doc.querySelector('.headline h1');
+		if (titleElem) {
+			item.title = titleElem.textContent.trim();
+		}
+
+		// Extract ISBN from MARC record
+		var marcText = doc.querySelector('#divmarc textarea');
+		if (marcText) {
+			var marcContent = marcText.textContent;
```

Comment:
Use the MARC translator instead of trying to parse MARC manually - see the Scaffold MARC template (Plus button -> Scrape Function Using -> MARC). You can translate the MARC record into a new item that you don't call `complete()` on if you just want to copy out certain fields.